src/java/org/apache/cassandra/tcm/transformations/cms/PrepareCMSReconfiguration.java

```diff
@@ -215,6 +215,9 @@ public long serializedSize(Transformation t, Version version)
 
     public static Diff diff(Set<NodeId> currentCms, Set<NodeId> newCms)
     {
+        assert !currentCms.contains(null);
+        assert !newCms.contains(null);
```

Comment:
nit: Could throw error messages in here, but it's not like we'd gain much information from that in debugging, unless we wanted to know what the rest of the nodes in the sets are as well...